client/my-sites/stats/stats-download-csv/index.jsx

```diff
@@ -89,7 +93,6 @@ class StatsDownloadCsv extends Component {
 		return (
 			<Button
 				className="stats-download-csv"
-				compact
```

Comment:
There is no `compact` prop on the Button component from WordPress core. [There is a `size="compact"` option](https://github.com/WordPress/gutenberg/blob/72c922b679cbc0d83220514165ea66837de131ca/packages/components/src/button/index.tsx#L153), but we don't seem to be using that anyway right now, so removing this does not seem to affect the display.

Removing this fixes the warning in the dev console

![Arc -2025-05-02 at 17 31 27@2x](https://github.com/user-attachments/assets/cb211742-87b4-4930-b37a-01a9d1958a20)